File: java-single/app/src/main/java/javaSingle/klv/common/enums/MeterFormat.java
package javaSingle.klv.common.enums;

public enum MeterFormat {
  SURROUND_STEREO_OR_MONO("Surround, stereo/mono"),
  STEREO_LO_RO("Stereo LoRo"),
  MS("M/S (sum, difference)"),
  STEREO_PHASE("Stereo phase"),
  MONO("Mono"),
  STEREO("Stereo"),
  SURROUND("Surround"),
  MF_UNDEFINED("undefined"),
  DOWNMIX_5_1("5.1 downmix");

  private final String legend;

  MeterFormat(final String legend) {
    this.legend = legend;
  }

  public String toString() {
    return legend;
  }

  public String getLegend() {
    return legend;
  }
}

File: python-single/src/klv/common/enums/meterFormat.py
from enum import IntEnum


class MeterFormat(IntEnum):
  SURROUND_STEREO_OR_MONO = 0
  STEREO_LO_RO = 1
  MS = 2
  STEREO_PHASE = 3
  MONO = 4
  STEREO = 5
  SURROUND = 6
  MF_UNDEFINED = 7
  DOWNMIX_5_1 = 8
